### PR TITLE
feat(storybook): Create stories for Avatar

### PR DESCRIPTION
**Pull request by laughnan**
This PR creates Storybook stories for `Avatar`.

Screen Shot 2020-01-12 at 16 02 08

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @laughnan has signed the CLA. Thanks for the pull request!

**Comment by laughnan** [comment]
can you give this a look @sahiga ??

File: src/components/avatar/Avatar.stories.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+export const withMultipleAvatars = () => (
+    <div>
+        <Avatar id={1} name="Aaron Levie" />
+        <Avatar id={2} name="Front End" />
+        <Avatar id={3} name="Redwood City" />
+    </div>
+);
+
+export const withoutNameOrInitials = () => <Avatar />;
```
Comment (sahiga) [inline src/components/avatar/Avatar.stories.js]: Can you add a test for this story as well?

Comment (laughnan) [reply src/components/avatar/Avatar.stories.js]: yes, of course. let me add that shortly.

**Pull request merged**